src/components/store/bestSellCategories/bestSellCategories.actions.ts

```diff
@@ -0,0 +1,19 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+export type GetBestSellCategoriesResponseData = {
```

Comment:
Consider separating your interfaces from actiions. For example you could create separate file to store your models

```
__shopItem
____shopItem.actions.ts
____shopItem.reducer.ts
____types.ts/model.ts --- all types/interfaces are stored here
```

Or you could go even further and separate your models by entities

```
__shopItem
____types.ts/model.ts -all types/interfaces are stored here
______shopItemsStore
________shopItems.reducer.ts 
...
______shopItemStore
________shopItem.reducer.ts
...
```

Reply:
This approach is valid too

```
__entities
____shopItem.ts -- all types/interfaces related to shop item
____shopCategory.ts -- all types/interfaces related to category

__store
___shopItem
____shopItem.reducer.ts -- import types from /entites
___shopCategory
____shopCategory.reducer.ts -- import types from /entities